metadata-ingestion/src/datahub/ingestion/reporting/datahub_ingestion_reporting_provider.py

```diff
@@ -29,21 +32,29 @@ class DatahubIngestionReportingProviderConfig(IngestionReportingProviderConfig):
 class DatahubIngestionReportingProvider(IngestionReportingProviderBase):
     orchestrator_name: str = "datahub"
 
-    def __init__(self, graph: DataHubGraph, name: str):
+    def __init__(self, graph: DataHubGraph, name: str, ctx: PipelineContext):
         super().__init__(name)
         self.graph = graph
         if not self._is_server_stateful_ingestion_capable():
             raise ConfigurationError(
                 "Datahub server is not capable of supporting stateful ingestion."
                 " Please consider upgrading to the latest server version to use this feature."
             )
+        # Initialize the UI hiostry aspect generator
```

Comment:
history?

Reply:
Could not come up with a better name :-( Any suggestions?

Reply:
no just typo^^